src/main/java/pl/stormit/tinyurl/controller/UrlAnalyticsController.java

```diff
@@ -0,0 +1,35 @@
+package pl.stormit.tinyurl.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.stormit.tinyurl.domain.model.UrlAnalytics;
+import pl.stormit.tinyurl.service.UrlAnalyticsService;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping(path = "api/v1/analytics")
+@RequiredArgsConstructor
+public class UrlAnalyticsController {
+
+    private final UrlAnalyticsService urlAnalyticsService;
+
+    @GetMapping("/{url-id}")
+    public ResponseEntity<List<UrlAnalytics>> getAnalyticsByUrlId(@NotNull @PathVariable("url-id") UUID urlId) {
```

Comment:
Why for list and single object, we have the same response? I thought that for SO we should have just one object.

Reply:
This response return a list of all analytics data about single shortUrl.
However, response allAnalytics return data about all shortUrl in database.
More information on Discord channel.

Reply:
Ok.